ciao-launcher/main.go

```diff
@@ -369,6 +385,23 @@ DONE:
 			if err != nil {
 				break DONE
 			}
+			clusterConfig, err := client.conn.ClusterConfiguration()
+			if err != nil {
+				glog.Errorf("Unable to get Cluster Configuration %v", err)
```

Comment:
@obedmr  Sorry, about this but I just noticed that we need to call

client.conn.Close()

here to shutdown the connection to the SSNTP server.